<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QMainWindow" name="Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1391</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное окно</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/main.ico</normaloff>../icons/main.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Main_Window{
background: gray;
}

QPushButton{
border-radius: 5px;
border: 1px solid black;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #fed5ff);
}

QPushButton:hover{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
}

QPushButton:pressed{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #55aa7f);
}

QLineEdit{
border-radius: 5px;
border: 1px solid black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="DB_Frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>1370</width>
      <height>431</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1371</width>
      <height>431</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#DB_Frame{
background: white;
border-radius: 50px;
border: 2px solid black;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="TableResidents">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1331</width>
       <height>391</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#TableResidents{
background: white;
border: 0px solid black;
}</string>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ФИО</string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Пол</string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Паспорт</string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Кем выдан</string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Когда выдан</string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Адресс регистрации</string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="TableStorage">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>1331</width>
       <height>391</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#TableStorage{
background: white;
border: 0px solid black;
}</string>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Наименование</string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Количество</string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Цена (руб.)</string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Минимальное количество</string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Еще что то </string>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="bttn_refresh">
     <property name="geometry">
      <rect>
       <x>1313</x>
       <y>17</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#bttn_refresh{
background: white;
border: 0px;
}

#bttn_refresh:hover{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
}

#bttn_refresh:pressed{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #55aa7f);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/refresh.ico</normaloff>../icons/refresh.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
    </widget>
    <zorder>TableStorage</zorder>
    <zorder>TableResidents</zorder>
    <zorder>bttn_refresh</zorder>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1371</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QFrame" name="Info_Frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>951</width>
         <height>101</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#Info_Frame_2{
background: white;
border-radius: 50px;
border: 2px solid black;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>921</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="bttn_create_form">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Создать форму</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bttn_parse">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Спарсить чо нить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Еще чо нить</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="Info_Frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>401</width>
         <height>101</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#Info_Frame{
background: white;
border-radius: 50px;
border: 2px solid black;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>341</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Режим использования</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_choice">
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>177</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>177</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Доступные БД</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboDatabase">
             <property name="font">
              <font>
               <family>Century Gothic</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>Жильцы</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Склад</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>570</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="bttn_add">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_del">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_change">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="bttn_exit">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>580</y>
      <width>109</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1391</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
